### PR TITLE
1362: Notification UX Refactor v1.0

### PR DESCRIPTION
**Pull request by AugusteBa**
notification17

### CODE REVIEW

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.js [hunk 1]
```diff
@@ -66,36 +68,22 @@ const Watchlist = () => {
...
-            </p>
-          </div>
-        </>
+        <div className="notification">
+          <div className="icon"></div>
+          <p>
+            {newSymbol.symbol}
+            {newSymbol.isNew
+              ? ' has been added to your Watchlist'
+              : ' has been moved to the top of your Watchlist'}
```
Comment (SL-SWoods) [inline packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: I wonder if its worth pulling out every instance where we have text like this into a single place? so that way if text ever needs changing it is only in a single place for every place?
Not part of this PR, but one for the backlog, as it could be useful for text like "no search results" etc.

Comment (AugusteBa) [reply packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: Yeah, a very good idea, that would be very nice indeed!

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.js [hunk 2]
```diff
@@ -45,18 +45,20 @@ const Watchlist = () => {
   const [options] = useOptions();
   const notification = useNotification(NOTIFICATION_OPTIONS);
 
-  const displayNotification = newSymbol => {
-    setNewSymbol(newSymbol);
+  const displayNotification = async () => {
     if (notification) {
-      notification.launch({
+      await notification.launch({
```
[the remark this answers is not in the document]
Comment (AugusteBa) [reply packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: Oh yeah, it is indeed necessary. Removed now, thanks for spotting this!

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.js [hunk 3]
```diff
@@ -0,0 +1,262 @@
+import React, { useState, useEffect } from 'react';
```
Comment (SL-SWoods) [inline packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: Should this file still be in the PR? it doesn't get referenced anymore, due to the refactor into app.js

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.css [hunk 4]
```diff
@@ -0,0 +1,103 @@
+* {
```
Comment (SL-SWoods) [inline packages/stockflux-watchlist/src/components/watchlist/Watchlist.css]: Should this file still be in the PR? it doesn't get referenced anymore, due to the refactor into app.js

File: packages/stockflux-watchlist/src/components/watchlist/Watchlist.js (hunk 3 above)
Comment (AugusteBa) [reply packages/stockflux-watchlist/src/components/watchlist/Watchlist.js]: Yeah, they can go! Thanks!
